spk/watcher3/src/wizard/upgrade_uifile

```diff
@@ -0,0 +1,6 @@
+[{
```

Comment:
That file can be deleted i guess you missed that when copying from CP ;)

Reply:
I were not sure what the purpose of this file is so I figured it would be best to keep it.